fish_prompt.fish

```diff
@@ -627,6 +628,16 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l __bobthefish_left_black_arrow_glyph  \uE0B2
   set -l __bobthefish_left_arrow_glyph        \uE0B3
 
+  if [ "$theme_powerline_fonts" = "no" ]
+    set __bobthefish_branch_glyph            ''
```

Comment:
What about ⎇ (U+2387, ALTERNATIVE KEY SYMBOL)?
However it's not present in all fonts, I believe.
(As a matter of fact I don't think it's important to show the branch icon in 'fallback' mode. It is fairly clear what the branch name means anyway.)